hathor/transaction/storage/transaction_storage.py

```diff
@@ -407,6 +401,7 @@ def get_best_block_tips(self, timestamp: Optional[float] = None, *, skip_cache:
             elif meta.score > best_score:
                 best_score = meta.score
                 best_tip_blocks = [block_hash]
+        self._best_block_tips = best_tip_blocks[:]
```

Comment:
Should it be `self. _best_block_tips_cache`?

Reply:
Sure! Great catch! Thanks!